### DISCUSSION TITLE
Character win sound for victory screen.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#1126] [Feature Request]
### Describe the solution you'd like

Similar to the parameter select.snd from character select screen which allows you to designate a sound for whenever you pick a character, with a additional time parameter that let you regulate how many ticks until the sound plays. I want this feature because I thought it would be cool to have something similar to KoF 96's victory screen. They could be named victory.snd and victory.snd.time.

### Additional context

https://youtu.be/ug8ms8nE0Ow?t=219

**Comment by potsmugen** [comment]
Probably no need for a SND file but it could play an MP3/whatever of your choice. Defined in the DEF file. I had to use a workaround to do this so this feature would be welcome (and not very hard to implement I think).

**Reply by UrsoManso** [reply]
I'm using the character select parameter as reference. You set the value on select.snd parameter and it will play whatever value is assigned from a character's snd file when you pick the character. Having to set a single mp3 file for every character would not be as pratical and also increase file size.

**Reply by potsmugen** [reply]
Actually that's just the thing. An MP3 file would be a lot smaller than a WAV inside a SND file.

**Comment by potsmugen** [comment]
I just realized this feature actually already exists, except you add the victory music through the select.def file.

I think it could be refactored so the music is loaded from the character's def file. Currently, if you want to distribute a char with a victory music included you'll have to provide instructions on how to install the char with music (or use workarounds like my chars do).

**Comment by UrsoManso** [comment]
I'm not talking about victory screen music, but the sound played on win states to play on victory screen. Something like this: https://youtu.be/DrkXUVmLcio?t=9


Enviado do Outlook<http://aka.ms/weboutlook>

________________________________
De: PotS ***@***.***>
Enviado: sexta-feira, 6 de outubro de 2023 11:47
Para: ikemen-engine/Ikemen-GO ***@***.***>
Cc: UrsoManso ***@***.***>; Author ***@***.***>
Assunto: Re: [ikemen-engine/Ikemen-GO] Character win sound for victory screen. (Discussion #1126)


I just realized this feature actually already exists, except you add the victory music through the select.def file.

I think it could be refactored so the music is loaded from the character's def file. Currently, if you want to distribute a char with a victory music included you'll have to provide instructions on how to install the char with music (or use workarounds like my chars do).

**Reply by potsmugen** [reply]
Oh, I see. I got you wrong from the start then, sorry. Should probably make a new thread for the music.
